src/wasm-traversal.h

```diff
@@ -208,8 +224,8 @@ struct Walker : public VisitorType {
     for (auto& curr : module->functions) {
       self->walkFunction(curr.get());
     }
-    self->visitTable(&module->table);
-    self->visitMemory(&module->memory);
+    self->walkTable(&module->table);
```

Comment:
Yeah, a walk will set things up, then call all the necessary visitors etc. The visitors do nothing by default and are meant for specific traversals to implement.